src/components/TransactionsTable/TransactionsTable.tsx

```diff
@@ -9,6 +9,7 @@ import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
 import { TxnType } from 'constants/index';
 import 'components/styles/TransactionsTable.scss';
+dayjs.extend(relativeTime);
```

Comment:
Why is this here?

Reply:
This is to call `fromNow` function of `dayjs`. Currently it is returning error on analytics page because we are missing this line